src/main/java/org/team1540/robot2019/OI.java

```diff
@@ -75,6 +81,8 @@
     // driver buttons
 
     public static JoystickButton fineDriveButton = new JoystickButton(driver, LB);
+    public static JoystickButton autoAlignButton = new JoystickButton(OI.driver, RB);
+    public static JoystickButton autoAlignCancelButton = new JoystickButton(OI.driver, LB);
```

Comment:
This clashes with `fineDriveButton`; can we find an alternate binding?

Reply:
The autoAlignCancelButton will be replaced with: if joysticks/triggers past deadzone -> stop auto-align

Reply:
Ya but we should def do that replacement pre-merge if at all possible.